src/js/bootstrap-slider.js

```diff
@@ -398,6 +398,21 @@ const windowIsDefined = (typeof window === "object");
 				}
 			}
 
+			function createTickMouseOverListener(reference, tick, index){
+				tick.addEventListener("mouseenter", function(){
+					var tempState = $.extend(true, reference._state, {});
+					var idString = index >= 0 ? index : $(this).attr('aria-valuenow');
```

Comment:
can you write this without JQuery? One of this libraries requirements is that it should work in environments without JQuery